docs/ENUM.md

```diff
@@ -405,7 +405,7 @@ To enable ENUM lookups on Sprout, edit `/etc/clearwater/shared_config` and add t
     enum_server=<IP addresses of enum servers>
     enum_file=<location of enum file>
 
-If you use the ENUM file, enter the ENUM rules in the JSON format (shown above).
+If you use the ENUM file, enter the ENUM rules in the JSON format (shown above).  If you use `/etc/clearwater/enum.json` as your ENUM filename, you can have it automatically synchronized across your deployment by running `sudo /usr/share/clearwater/clearwater-config-manager/scripts/upload_enum_json` after creating or updating the file.  In this case, other Sprout nodes will automatically download and use the uploaded ENUM rules.
```

Comment:
We should make it clear that this is only available if you're using clearwater-etcd for configuration.